### ISSUE TITLE
Unknown Horizons

### ISSUE THREAD

**Issue by 7wells** [Calinou/scoop-games#144]
Hello Calinou,

Could you please consider adding UH to the bucket? The link leads to the Fifengine version:
https://github.com/unknown-horizons/unknown-horizons

I am not sure whether the following is also needed:
https://github.com/unknown-horizons/gfx
https://github.com/unknown-horizons/gfx

Please note that Unknown-Horizons based on Fifengine is no longer in development. They are porting it to the Godot engine. Is this a reason to not add UH with the Fifengine?

**Comment by Calinou** [comment]
> Please note that Unknown-Horizons based on Fifengine is no longer in development. They are porting it to the Godot engine. Is this a reason to not add UH with the Fifengine?

We can still add the Fifengine-based version as it will remain the most full-featured version of the game for the foreseeable future :slightly_smiling_face:

**Comment by 7wells** [comment]
Yes, indeed. 😉 As far as I can see, they are rather at the beginning of porting it to the Godot engine. I would like to help here and there but am myself at the very beginning of grasping the basic concepts of coding etc. Sorry, this gets too OT here. 😊 Thanks for adding the Fifengine UH version. If there is anything easy for me to do and to learn from it, please consider letting me know.

**Comment by beyondmeat** [comment]
@7wells close?

**Comment by Calinou** [comment]
I think this is still relevant to add. Feel free to open a pull request for it :slightly_smiling_face:
